topics/proteomics/tutorials/clinical-mp-quantitation/tutorial.md

```diff
@@ -0,0 +1,215 @@
+
+The outputs we are most interested in consist of the `MaxQuant Evidence file`, `MaxQuant Protein Group`s, and `MaxQuant Peptides`. The `MaxQuant Peptides` file will allow us to group them to generate a list of quantified microbial peptides.
+
+> <hands-on-title> Quantify verified peptides (from PepQuery2) </hands-on-title>
+>
+> 1. {% tool [MaxQuant](toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant/1.6.17.0+galaxy4) %} with the following parameters:
+>    - In *"Input Options"*:
+>        - {% icon param-file %} *"FASTA files"*: `Quantitation Database for MaxQuant` (Input dataset)
+>    - In *"Search Options"*:
+>        - {% icon param-file %} *"Specify an experimental design template (if needed). For detailed                           instructions see the help text."*: `output` (Input dataset)
+>        - *"minimum peptide length"*: `8`
+>        - *"Match between runs"*: `Yes`
+>        - *"Maximum peptide length for unspecific searches"*: `50`
+>    - In *"Protein quantification"*:
+>        - *"Use only unmodified peptides"*: `Yes`
+>            - *"Modifications used in protein quantification"*: `Oxidation (M)`
+>        - In *"LFQ Options"*:
+>            - *"iBAQ (calculates absolute protein abundances by normalizing to copy number and not protein mass)"*: `No`
+>    - In *"Parameter Group"*:
+>        - {% icon param-repeat %} *"Insert Parameter Group"*
+>            - {% icon param-collection %} *"Infiles"*: `output` (Input dataset collection)
+>            - *"fixed modifications"*: `Carbamidomethyl (C)`
+>            - *"variable modifications"*: `Oxidation (M)`
+>            - *"enzyme"*: `Trypsin/P`
+>            - *"Quantitation Methods"*: `reporter ion MS2`
+>                - *"isobaric labeling"*: `TMT11plex`
+>                - *"Filter by PIF"*: `Yes`
+>    - In *"Output Options"*:
+>        - *"Select the desired outputs."*: `Protein Groups` `mqpar.xml` `Peptides` `Evidence` `MSMS`
+>
+>
+{: .hands_on}
+
+
+> <question-title></question-title>
+>
+> 1. Why can we switch back to using RAW files for MaxQuant, instead of using MGF files?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. MaxQuant prefers RAW format compared to MGF as it has more information compared to MGF. 
+> >
+> {: .solution}
+>
+{: .question}
+> <question-title></question-title>
+>
+> 1. Previously, we used MaxQuant in the Discovery workflow. Why are we using MaxQuant again, instead of Search GUI/PeptideShaker?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. We are using MaxQuant for quantification purposes only. SearchGUI Peptide Shaker doesn't have the capability to perform quantification of peptides or proteins. 
+> >
+> {: .solution}
+>
+{: .question}
+
+## Using Text Manipulation Tools to Manage MaxQuant Outputs
```

Comment:
This is a good section title, I would just group all the steps in a single hands-on box